include/oboe/Definitions.h

```diff
@@ -155,6 +155,13 @@ namespace oboe {
         AAudio
     };
 
+// Help compile with old NDK versions.
```

Comment:
Whilst I understand the reasoning behind this approach it's not particularly readable. My preference would be to have a separate `AAudioDefinitions.h` file which is a copy of the AAudio defines from later NDK versions.

Reply:
I'm not clear what you mean. Would we include a different file based on API_LEVEL?

Maybe just define the enums twice
#if API_LEVEL
    enum { GAMES = 4, /* AAUDIO_GAMES */ }
#else 
    enum { GAMES = AAUDIO_GAMES, }
#endif 

It's verbose but readable.